wolox-books/src/app/register/register.component.html

```diff
@@ -0,0 +1,34 @@
+<form [formGroup]="rForm" (ngSubmit)="addPost(rForm.value)" class="form">
+  <div class="form-container">
```

Comment:
I think this `div` is not necessary. Use the `form-container` class in the `form` tag